region/lowernorfair/east/Metal Pirates Room.json

```diff
@@ -253,6 +253,244 @@
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"heatFrames": 125},
+        {"or": [
+          {"and": [
+            "Charge", "Plasma",
+            "canDodgeWhileShooting",
+            "canTrickyJump",
+            {"or": [
+              {"and": [
+                "Ice", {"heatFrames": 700}
+              ]},
+              {"and": [
+                "Wave", {"heatFrames": 700}
+              ]},
+              {"and": [
+                "canInsaneJump",
+                "Ice", {"heatFrames": 520}
+              ]},
+              {"and": [
+                "canInsaneJump",
+                "Wave", {"heatFrames": 520}
+              ]},
+              {"and": [
+                "Ice", "Wave", {"heatFrames": 400}
+              ]},
+              {"heatFrames": 950},
+              {"and": [
+                "canInsaneJump",
+                {"heatFrames": 700}
+              ]},
+              {"and": [
+                "canXRayWaitForIFrames",
+                {"heatFrames": 420}
+              ]}
+            ]}
+          ]},
+          {"and": [
+            "Charge", "Spazer",
+            "canDodgeWhileShooting",
+            {"or": [
+              {"heatFrames": 4400},
+              {"and": [
+                "Ice", {"heatFrames": 3000}
+              ]},
+              {"and": [
+                "Wave", {"heatFrames": 2650}
+              ]},
+              "Ice", "Wave", {"heatFrames": 1750}
+            ]}
+          ]},
+          {"and": [
+            "Plasma",
+            "canDodgeWhileShooting",
+            {"heatFrames": 2000}
+          ]},
+          {"and": [
+            "h_heatProof",
+            "canDodgeWhileShooting",
+            {"or": [
+              {"and": ["Charge", "Wave"]},
+              {"and": ["Charge", "Ice", "canBePatient"]},
+              {"and": ["canBeVeryPatient", {"or": ["Charge", "Spazer"]}]}
+            ]}
+          ]},
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyKill": {"enemies": [["Space Pirate (fighting)", "Space Pirate (fighting)"]], "explicitWeapons": ["Super"]}},
+            {"heatFrames": 450}
+          ]},
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyKill": {"enemies": [["Space Pirate (fighting)", "Space Pirate (fighting)"]], "explicitWeapons": ["Missile"]}},
+            {"heatFrames": 2700}
+          ]}
+        ]},
+        {"or": [
+          "h_heatedCrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 240}}
+              ]},
+              {"partialRefill": {"type": "ReserveEnergy", "limit": 120}}
+            ]}
+          ]},
+          {"and": [
+            {"disableEquipment": "ETank"},
```

Comment:
Collecting drops from both pirates would take either
- Varia
- One round each pirate with supers or Plasma + X
- split your damage by switching which pirate you are damaging so they die at the same time?